.eslintrc

```diff
@@ -51,6 +51,16 @@
     "@babel/implicit-arrow-linebreak": "off",
     "@babel/array-bracket-newline": "off",
     "@babel/brace-style": "off",
-    "@babel/semi": "warn"
-  }
+    "@babel/semi": "warn",
+    // `any` is useful for incremental type improvements during the transition from JS to TS.
+    "@typescript-eslint/no-explicit-any": "off",
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {
+        "no-undef": "off"
+      }
+    }
+  ]
 }
```

Comment:
Side note - I'd really like this kind of thing to be in a separate PR so we can benefit from it sooner.